src/chitterApi.js

```diff
@@ -0,0 +1,103 @@
+class chitterApi {
+  constructor() {
+    this.apiUrl = "https://chitter-backend-api-v2.herokuapp.com/peeps";
```

Comment:
no need to repeat `"https://chitter-backend-api-v2.herokuapp.com/"`. Instead, store it once as a constant or in a config file as base_url. This way, when the server changes address you only have to change it once.

Reply:
I totally agree